src/references/uniswap/index.ts

```diff
@@ -37,20 +34,25 @@ const ETHER_WITH_ADDRESS: RainbowToken = {
   uniqueId: 'eth',
 };
 
-const TOKEN_LIST_WITH_ETH: RainbowToken[] = [ETHER_WITH_ADDRESS, ...TOKEN_LIST];
+const tokenListWithEth: RainbowToken[] = [ethWithAddress, ...tokenList];
 
 const RAINBOW_TOKEN_LIST: Record<string, RainbowToken> = keyBy(
-  TOKEN_LIST_WITH_ETH,
+  tokenListWithEth,
   'address'
 );
 
-const CURATED_RAINBOW_TOKEN_LIST: RainbowToken[] = filter(
-  TOKEN_LIST_WITH_ETH,
+const curatedRainbowTokenList: RainbowToken[] = filter(
+  tokenListWithEth,
   'isRainbowCurated'
 );
 
+const TOKEN_SAFE_LIST: Record<string, string> = keyBy(
```

Comment:
This is a map where the keys are the symbols and names of the curated list which is used for the scam check.